### PR TITLE
Regroup list of public repos

### PR DESCRIPTION
**Pull request by descorp**
# Open PR

## Changes

* add `adyen-networking-ios`
* add `adyen-3ds2-ios`
* add `adyen-3ds2-android`
* rename `Adyen React Native [Beta]` -> `Adyen React Native [BETA]`
* regroup everything
* remove `Adyen` everywhere
* convert all labels to **sentence case**
* mark "API library for Apex" and "PHP webhook helper module for payment integrations" as **BETA**
* mark "Plugin for feast to run an offline store on Spark" as **Discontinued**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -26,35 +26,56 @@ This page contains the source code for all of our libraries, SDKs and plug-ins a
...
 * [Adyen Terminal API for iOS](https://github.com/Adyen/adyen-terminal-api-ios)
 
-### Plug-ins
+### Payment plugins
 
-* [Adyen Payment plugin for Magento2](https://github.com/Adyen/adyen-magento2) 
-* [Adyen API Library for Apex](https://github.com/Adyen/adyen-apex-api-library)
 * [Adyen Payment plugin for Shopware 6](https://github.com/Adyen/adyen-shopware6)
 * [Adyen Payment plugin for Shopware 5](https://github.com/Adyen/adyen-shopware5)
+* [Adyen Payment plugin for Magento2](https://github.com/Adyen/adyen-magento2)
```
Comment (candemiralp) [inline profile/README.md]: Can we move Magento 2 on top of the Shopware 6?

File: profile/README.md [hunk 2]
```diff
@@ -26,35 +26,56 @@ This page contains the source code for all of our libraries, SDKs and plug-ins a
...
-* [Adyen iOS Drop-in and Components](https://github.com/Adyen/adyen-ios)
-* [Adyen Android Drop-in and Components](https://github.com/Adyen/adyen-android)
+* [Adyen API Library for Apex [BETA]](https://github.com/Adyen/adyen-apex-api-library)
+
+#### Terminal API
+
 * [Adyen Terminal API for iOS](https://github.com/Adyen/adyen-terminal-api-ios)
 
-### Plug-ins
+### Payment plugins
```
Comment (candemiralp) [inline profile/README.md]:
```suggestion
### Payment Plugins
```

File: profile/README.md [hunk 3]
```diff
@@ -26,35 +26,56 @@ This page contains the source code for all of our libraries, SDKs and plug-ins a
...
 
-* [Adyen Payment plugin for Magento2](https://github.com/Adyen/adyen-magento2) 
-* [Adyen API Library for Apex](https://github.com/Adyen/adyen-apex-api-library)
 * [Adyen Payment plugin for Shopware 6](https://github.com/Adyen/adyen-shopware6)
 * [Adyen Payment plugin for Shopware 5](https://github.com/Adyen/adyen-shopware5)
+* [Adyen Payment plugin for Magento2](https://github.com/Adyen/adyen-magento2) 
+* [Adyen Payment plugin for Prestashop](https://github.com/Adyen/adyen-prestashop)
+* [Adyen Payment plugin for Hybris](https://github.com/Adyen/adyen-hybris)
+
+### Integration plugins
```
Comment (candemiralp) [inline profile/README.md]:
```suggestion
### Integration Plugins
```

File: profile/README.md [hunk 4]
```diff
@@ -26,35 +26,56 @@ This page contains the source code for all of our libraries, SDKs and plug-ins a
...
+
 * [Salesforce Commerce Cloud (formerly Demandware)](https://github.com/Adyen/adyen-salesforce-commerce-cloud)
 * [Adyen plugin for Oracle Commerce Cloud](https://github.com/Adyen/adyen-oracle-commerce-cloud)
 * [Adyen app for Salesforce B2B Commerce](https://github.com/Adyen/adyen-salesforce-b2b-commerce)
-* [Adyen Payment plugin for Prestashop](https://github.com/Adyen/adyen-prestashop)
-* [Adyen Payment plugin for Hybris](https://github.com/Adyen/adyen-hybris)
 
-### Others
+### Others repos
```
Comment (candemiralp) [inline profile/README.md]:
```suggestion
### Other Repos
```

File: profile/README.md [hunk 5]
```diff
@@ -26,35 +26,56 @@ This page contains the source code for all of our libraries, SDKs and plug-ins a
...
+### Integration plugins
+
 * [Salesforce Commerce Cloud (formerly Demandware)](https://github.com/Adyen/adyen-salesforce-commerce-cloud)
 * [Adyen plugin for Oracle Commerce Cloud](https://github.com/Adyen/adyen-oracle-commerce-cloud)
 * [Adyen app for Salesforce B2B Commerce](https://github.com/Adyen/adyen-salesforce-b2b-commerce)
-* [Adyen Payment plugin for Prestashop](https://github.com/Adyen/adyen-prestashop)
-* [Adyen Payment plugin for Hybris](https://github.com/Adyen/adyen-hybris)
 
-### Others
+### Other repos
```
Comment (candemiralp) [inline profile/README.md]:
```suggestion
### Other Repos
```

**Review by michaelpaul — APPROVED** [review]

File: profile/README.md (hunk 1 above)
Comment (descorp) [reply profile/README.md]: fixed in [bbac6d7](https://github.com/Adyen/.github/pull/7/commits/bbac6d73fc6f0a41baf16258b2a3ac188c4a4185)

File: profile/README.md (hunk 2 above)
Comment (descorp) [reply profile/README.md]: fixed in [bbac6d7](https://github.com/Adyen/.github/pull/7/commits/bbac6d73fc6f0a41baf16258b2a3ac188c4a4185)

File: profile/README.md (hunk 3 above)
Comment (descorp) [reply profile/README.md]: fixed in [bbac6d7](https://github.com/Adyen/.github/pull/7/commits/bbac6d73fc6f0a41baf16258b2a3ac188c4a4185)

File: profile/README.md (hunk 4 above)
Comment (descorp) [reply profile/README.md]: fixed in [bbac6d7](https://github.com/Adyen/.github/pull/7/commits/bbac6d73fc6f0a41baf16258b2a3ac188c4a4185)

File: profile/README.md (hunk 2 above)
Comment (AdrienneBrieffies) [reply profile/README.md]: At Adyen we use sentence case for headings and titles ("Payment plugins"), not title case ("Payment Plugins").

File: profile/README.md (hunk 4 above)
Comment (AdrienneBrieffies) [reply profile/README.md]: Please use sentence case ("Other repos") and consider using the full word: Other repositories

File: profile/README.md (hunk 5 above)
Comment (descorp) [reply profile/README.md]: Duplicate

**Review by wboereboom — APPROVED** [review]

File: profile/README.md [hunk 6]
```diff
@@ -26,35 +26,56 @@ This page contains the source code for all of our libraries, SDKs and plug-ins a
...
+### Integration Plugins
+
 * [Salesforce Commerce Cloud (formerly Demandware)](https://github.com/Adyen/adyen-salesforce-commerce-cloud)
 * [Adyen plugin for Oracle Commerce Cloud](https://github.com/Adyen/adyen-oracle-commerce-cloud)
 * [Adyen app for Salesforce B2B Commerce](https://github.com/Adyen/adyen-salesforce-b2b-commerce)
-* [Adyen Payment plugin for Prestashop](https://github.com/Adyen/adyen-prestashop)
-* [Adyen Payment plugin for Hybris](https://github.com/Adyen/adyen-hybris)
 
-### Others
+### Other Repos
```
Comment (rikterbeek) [inline profile/README.md]:
```suggestion
### Other repos
```

File: profile/README.md [hunk 7]
```diff
@@ -26,35 +26,56 @@ This page contains the source code for all of our libraries, SDKs and plug-ins a
...
-* [Adyen iOS Drop-in and Components](https://github.com/Adyen/adyen-ios)
-* [Adyen Android Drop-in and Components](https://github.com/Adyen/adyen-android)
+* [Adyen API Library for Apex [BETA]](https://github.com/Adyen/adyen-apex-api-library)
+
+#### Terminal API
+
 * [Adyen Terminal API for iOS](https://github.com/Adyen/adyen-terminal-api-ios)
 
-### Plug-ins
+### Payment Plugins
```
Comment (rikterbeek) [inline profile/README.md]:
```suggestion
### Payment plugins
```

File: profile/README.md [hunk 8]
```diff
@@ -26,35 +26,56 @@ This page contains the source code for all of our libraries, SDKs and plug-ins a
...
+* [Wrapper for 3DS 2.0 library for iOS](https://github.com/Adyen/adyen-3ds2-ios)
+* [Wrapper for 3DS 2.0 library for Android](https://github.com/Adyen/adyen-3ds2-android)
+* [Helper functions to get 3DS 2.0 integrated on the front-end](https://github.com/Adyen/adyen-3ds2-js-utils)
+
+#### Supplementary
+
+* [Adyen Networking library for iOS](https://github.com/Adyen/adyen-networking-ios)
+* [Adyen WeChat Pay SDK Wrapper for iOS](https://github.com/Adyen/adyen-wechatpay-ios)
+
+#### API Libraries
```
Comment (rikterbeek) [inline profile/README.md]:
```suggestion
#### API libraries
```

File: profile/README.md [hunk 9]
```diff
@@ -26,35 +26,56 @@ This page contains the source code for all of our libraries, SDKs and plug-ins a
 
 ### Libraries and SDKs
 
-* [Adyen Web Drop-in and Components](https://github.com/Adyen/adyen-web)
+#### Drop-in and Components
```
Comment (rikterbeek) [inline profile/README.md]:
```suggestion
#### Drop-in and components
```

File: profile/README.md [hunk 10]
```diff
@@ -26,35 +26,56 @@ This page contains the source code for all of our libraries, SDKs and plug-ins a
...
+### Payment Plugins
 
 * [Adyen Payment plugin for Magento2](https://github.com/Adyen/adyen-magento2) 
-* [Adyen API Library for Apex](https://github.com/Adyen/adyen-apex-api-library)
 * [Adyen Payment plugin for Shopware 6](https://github.com/Adyen/adyen-shopware6)
 * [Adyen Payment plugin for Shopware 5](https://github.com/Adyen/adyen-shopware5)
+* [Adyen Payment plugin for Prestashop](https://github.com/Adyen/adyen-prestashop)
+* [Adyen Payment plugin for Hybris](https://github.com/Adyen/adyen-hybris)
+
+### Integration Plugins
```
Comment (rikterbeek) [inline profile/README.md]:
```suggestion
### Integration plugins
```

Comment (descorp) [reply profile/README.md]: Fixed on [cab4990](https://github.com/Adyen/.github/pull/7/commits/cab4990013fbcba82db1356b6602f758dec59390)

File: profile/README.md (hunk 9 above)
Comment (descorp) [reply profile/README.md]: Fixed on [cab4990](https://github.com/Adyen/.github/pull/7/commits/cab4990013fbcba82db1356b6602f758dec59390)

File: profile/README.md (hunk 8 above)
Comment (descorp) [reply profile/README.md]: Fixed on [cab4990](https://github.com/Adyen/.github/pull/7/commits/cab4990013fbcba82db1356b6602f758dec59390)

File: profile/README.md (hunk 7 above)
Comment (descorp) [reply profile/README.md]: Fixed on [cab4990](https://github.com/Adyen/.github/pull/7/commits/cab4990013fbcba82db1356b6602f758dec59390)

File: profile/README.md (hunk 6 above)
Comment (descorp) [reply profile/README.md]: Fixed on [cab4990](https://github.com/Adyen/.github/pull/7/commits/cab4990013fbcba82db1356b6602f758dec59390)

File: profile/README.md (hunk 4 above)
Comment (descorp) [reply profile/README.md]: Fixed as 'Other repos' on [cab4990](https://github.com/Adyen/.github/pull/7/commits/cab4990013fbcba82db1356b6602f758dec59390)

File: profile/README.md (hunk 3 above)
Comment (descorp) [reply profile/README.md]: Fixed on [cab4990](https://github.com/Adyen/.github/pull/7/commits/cab4990013fbcba82db1356b6602f758dec59390)

File: profile/README.md (hunk 2 above)
Comment (descorp) [reply profile/README.md]: Fixed on [cab4990](https://github.com/Adyen/.github/pull/7/commits/cab4990013fbcba82db1356b6602f758dec59390)

**Review by michaelpaul — APPROVED** [review]

**Comment by rikterbeek** [comment]
HI @descorp,

Are you done with the changes if so can you remove the [WIP].

Regards,
Rik

**Comment by descorp** [comment]
> HI @descorp,
> 
> Are you done with the changes if so can you remove the [WIP].
> 
> Regards, Rik

Waiting for review from 

| Team | Approve |
| --- | --- |
| (Tech Writers) | ✅ |
| @Adyen/integration-tools | ✅ | 
| @Adyen/plugins-partners | ✅ | 
| @Adyen/ipp-mobile-ios | ✅ |

**Review by kaphacius — APPROVED** [review]

**Review by Morerice — APPROVED** [review]

**Review by maassenbas — APPROVED** [review]

**Review by peterojo — APPROVED** [review]

**Pull request merged**